Meadowlark-js/packages/meadowlark-core/test/metaed/MetaEdValidation.test.ts

```diff
@@ -5,50 +5,49 @@
 
 import { newTopLevelEntity, newEntityProperty } from '@edfi/metaed-core';
 import Joi from '@hapi/joi';
-import { validatePartialEntityBodyAgainstSchema } from '../../src/metaed/MetaEdValidation';
+import { validateQueryParametersAgainstSchema } from '../../src/metaed/MetaEdValidation';
 
-const createModel = () => {
-  const model = newTopLevelEntity();
-  model.baseEntityName = 'Student';
-  const property = newEntityProperty();
-  property.metaEdName = 'uniqueId';
-  model.properties.concat([property]);
-  model.data.meadowlark = {
-    joiSchema: Joi.object({
-      uniqueId: Joi.string(),
-    }),
-  };
-  return model;
-};
+const createModel = () => ({
```

Comment:
I changed this to create the object as an object literal, using object spread where needed. A key to doing it this way with an arrow function is to wrap it in parentheses so JS can differentiate the object literal from a code block.